plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -959,12 +959,16 @@
 }
 
 #variable_product_options #message,
+#inventory_product_data .notice,
 #variable_product_options .notice {
 	display: flex;
 	margin: 10px;
 	background-color: #ffffff;
 	> p {
 		width: 85%;
```

Comment:
This `width: 85%` results in the notice looking odd when it goes to two or more lines.

I think it can just be removed.

Before (made the text longer to simulate a longer translation or different user settings that would result in more lines):

<img width="850" alt="Screenshot 2023-03-13 at 09 23 26" src="https://user-images.githubusercontent.com/2098816/224714888-23a96130-111e-40cc-81f6-b132a8ec60c5.png">



After:

<img width="850" alt="Screenshot 2023-03-13 at 09 24 10" src="https://user-images.githubusercontent.com/2098816/224715017-ed040639-b621-44d0-814a-16877edc424a.png">

Reply:
Nice catch! I couldn't delete it because we will need the 85% when there is a button, like here:

![Screenshot 2023-03-13 at 11 48 43](https://user-images.githubusercontent.com/1314156/224737498-0000aaf8-e6e0-44ed-a0a9-3086652099de.png)

But I added the code we need for that case in the commit [5108e42](https://github.com/woocommerce/woocommerce/pull/37185/commits/5108e42f2bf3b391a326a390112ae19ff3b83828).

Reply:
That explicit 85%/15% sizing is not ideal, and results in display issues. But, that's really outside the scope of this PR I suppose... I'll create a separate PR later today to fix that up...

Before:

<img width="641" alt="Screenshot 2023-03-13 at 11 50 57" src="https://user-images.githubusercontent.com/2098816/224755908-97ea3960-967f-4e0f-8aca-cac95566a1f1.png">

After removing explicit widths:

<img width="641" alt="Screenshot 2023-03-13 at 11 53 09" src="https://user-images.githubusercontent.com/2098816/224755949-296bf15d-1cc0-479f-84ee-4eeaa006f22f.png">